src/background/NotificationWindow/createBrowserWindow.ts

```diff
@@ -66,19 +63,38 @@ export async function createBrowserWindow({
   } else {
     heightValue = height;
   }
+  const url = makePopupRoute(`${initialRoute}?${params.toString()}`, type);
 
-  const { id: windowId } = await browser.windows.create({
-    focused: true,
-    url: makePopupRoute(`${initialRoute}?${params.toString()}`, type),
-    type: type === 'dialog' ? 'popup' : 'normal',
-    width,
-    height: heightValue,
-    ...position,
-  });
+  let window: Windows.Window | undefined;
+  try {
+    window = await browser.windows.create({
+      focused: true,
+      url,
+      type: type === 'dialog' ? 'popup' : 'normal',
+      width,
+      height: heightValue,
```

Comment:
maybe let's extract this to something like `windowOptions` and reuse in `catch`?

Reply:
I thought about this. Let me try to do it